llm/predictor.py

```diff
@@ -732,7 +760,7 @@ def predict():
                 target_texts.append(example["tgt"])
     else:
         source_texts = ["hello world, how are you?", "你好，请问你是谁?"]
-        target_texts = ["", ""]
+        target_texts = ["", "", ""]
```

Comment:
source_texts 的长度为 2， target_texts 的长度应该也是 2，所以这样代码删掉呗。

```suggestion
        target_texts = ["", ""]
```